src/js/mockedData.jsx

```diff
@@ -131,94 +131,105 @@ const REASON_CODE_MOCKS = [
   { value: '3', label: 'Because I said so' },
 ];
 
-const LOT_DATA = [
+const AVAILABLE_LOTS = [
   {
-    productCode: 1,
+    product: { code: 1, name: 'Advil 200mg' },
```

Comment:
Actually we should keep this as productCode as that's what will be returned by the API. And note that it's a string, not an integer.

Reply:
I made this change, to have same approach as on mocks from previous steps. I also assumed, that once we will have API, then we will need to make some small adjustments. However this can be easily changed in future.

Reply:
Awesome. And yes I figured the same (re: small adjustments). I'm just keeping an eye out to make sure I don't add too much burden to that effort once the APIs are actually ready.